FileManager/App/AppDelegate.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  AppDelegate.swift
+//  FileManager
+//
+//  Created by Екатерина Неделько on 31.05.22.
+//
+
+import UIKit
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        
+        if #available(iOS 15, *) {
```

Comment:
Your application minimum version is 15.4. There's no need for ` if #available(iOS 15, *)`